include/grpc++/server.h

```diff
@@ -57,24 +61,27 @@ class ServerAsyncStreamingInterface;
 class ServerCredentials;
 class ThreadPoolInterface;
 
-// Currently it only supports handling rpcs in a single thread.
 class Server GRPC_FINAL : public GrpcLibrary, private CallHook {
```

Comment:
Probably we want some docstring for Server...

represents a server
created by ServerBuilder